### PR TITLE
Add unit test for pkg/serviceaccount.

### PR DESCRIPTION
**Pull request by tizhou86**
**What this PR does / why we need it**:
Add Unit test for pkg/serviceaccount.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
None

**Special notes for your reviewer**:
None

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/serviceaccount/util_test.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2014 The Kubernetes Authors.
```
Comment (dixudx) [inline pkg/serviceaccount/util_test.go]: s/2014/2018

File: pkg/serviceaccount/util_test.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	}
+
+	for k, v := range tests {
+		actual := IsServiceAccountToken(v.secret, v.sa)
+		if actual != v.expect {
+			t.Errorf("%s failed, expected %t but received %t", k, v.expect, actual)
+		}
+	}
+	
+}
```
Comment (dixudx) [inline pkg/serviceaccount/util_test.go]: add a newline here.

**Review by dixudx — COMMENTED** [review]
small nits.

**Review by drinktee — APPROVED** [review]

File: pkg/serviceaccount/util_test.go (hunk 2 above)
Comment (tizhou86) [reply pkg/serviceaccount/util_test.go]: Fixed.

**Comment by k82cn** [comment]
please merge the commit :).

**Comment by tizhou86** [comment]
@k82cn Squashed, thanks!

**Pull request merged**